x-pack/plugins/lens/public/app_plugin/csv_download_provider/csv_download_provider.tsx

```diff
@@ -174,9 +167,8 @@ export const downloadCsvShareProvider = ({
               generateExportButton: (
                 <FormattedMessage id="xpack.lens.share.csvButton" defaultMessage="Download CSV" />
```

Comment:
I noticed that the text of the button says "Download CSV", if the license is less-than `gold`. This happens in `main` as well. I think the intention was that the button would say "Download CSV" any time the CSV option is selected